### PR TITLE
Optional value should only be accessed after calling isPresent()

### PR DESCRIPTION
**Pull request by zhaoyuguang**
### Describe what this PR does / why we need it
Optional value should only be accessed after calling isPresent()

### CODE REVIEW

**Comment by fangjian0423** [comment]
hi, could you refer https://github.com/alibaba/spring-cloud-alibaba/wiki/%E7%89%88%E6%9C%AC%E8%AF%B4%E6%98%8E to format code.

**Comment by zhaoyuguang** [comment]
> hi, could you refer https://github.com/alibaba/spring-cloud-alibaba/wiki/%E7%89%88%E6%9C%AC%E8%AF%B4%E6%98%8E to format code?

Is it? spring-cloud-alicloud-oss/src/main/java/com/alibaba/alicloud/oss/resource/OssStorageResource.java

**Comment by zhaoyuguang** [comment]
I solved it

File: spring-cloud-alicloud-oss/src/main/java/com/alibaba/alicloud/oss/resource/OssStorageResource.java [hunk 1]
```diff
@@ -155,9 +156,9 @@ public InputStream getInputStream() throws IOException {
 	 * @return the bucket if it exists, or null otherwise
 	 */
 	public Bucket getBucket() {
-		return this.oss.listBuckets().stream()
-				.filter(bucket -> bucket.getName().equals(this.bucketName)).findFirst()
-				.get();
+		Optional<Bucket> value = this.oss.listBuckets().stream()
+				.filter(bucket -> bucket.getName().equals(this.bucketName)).findFirst();
+		return value.orElse(null);
```
Comment (fangjian0423) [inline spring-cloud-alicloud-oss/src/main/java/com/alibaba/alicloud/oss/resource/OssStorageResource.java]: `return this.oss.listBuckets().stream().filter(bucket -> bucket.getName().equals(this.bucketName)).findFirst().orElse(null)` is better.

Comment (zhaoyuguang) [reply spring-cloud-alicloud-oss/src/main/java/com/alibaba/alicloud/oss/resource/OssStorageResource.java]: Yes, thank for your guidance. 😀

**Pull request merged**